provisioning/local/azure.tf

```diff
@@ -0,0 +1,64 @@
+provider "azurerm" {
+  tenant_id       = "9b85ee6f-4fb0-4a46-8cb7-4dcc6b262a89" // Keboola
+  subscription_id = var.azure_subscription_id
+  features {}
+}
+
+locals {
+  location = "West Europe"
+}
+
+variable "azure_subscription_id" {
+  type = string
+  validation {
+    condition     = length(var.azure_subscription_id) > 0
+    error_message = "The \"azure_subscription_id\" must be non-empty string."
+  }
+}
+
+resource "azurerm_resource_group" "api_error_control_resource_group" {
+  location = local.location
+  name     = "${var.name_prefix}-api-error-control"
+}
```

Comment:
Tady, podobne jako u AWS, takova ciste osobni preference, nedavat tu `location` do promenne, ale nadratoval bych ji do te resource group a pak u ostatni resources referencoval region te resource group. unguje to univerzalneji treba v situaci, kdyby si mel tech resource group vic.